agfusion/parsers.py

```diff
@@ -2,8 +2,10 @@
 Parses output files from fusion-finding algorithms
 """
 
+import os
 import re
 import csv
+import zipfile
```

Comment:
We can probably remove this one too!

Reply:
Just took it out, thanks for catching that!